.github/workflows/ms.compute.virtualmachines.yml

```diff
@@ -27,10 +27,10 @@ on:
     branches:
       - main
     paths:
-      - 'modules/compute/virtual-machines/**'
-      - 'modules/recovery-services/vaults/protection-containers/protected-items/**'
-      - 'modules/network/public-ip-addresses/**'
-      - 'modules/network/network-interfaces/**'
+      - 'modules/compute/virtual-machine/**'
+      - 'modules/recovery-services/vault/protection-containers/protected-items/**'
```

Comment:
- Please merge latest from `users/avm/3452_singularNames` (just merged latest from main)
- Cross references miss conversion to singular for child module folders, e.g. here should be
`modules\recovery-services\vaults\backup-fabric\protection-container\protected-item`
(Please check other pipelines)

Reply:
Merged and additional references updated